### ISSUE TITLE
[charter] What is a specification as currently defined in the charter

### ISSUE THREAD

**Issue by karlcow** [web-platform-tests/interop#121]
This is an issue opened because of the [pending review on the charter draft](https://github.com/web-platform-tests/interop/pull/102).

Do we need a table for each organizations on what is standard material or not.

**Comment by foolip** [comment]
I'd like us to roughly match the approach from WPT, where we accept tests for anything that claims to be a spec, that has implementation requirements for browsers. We're limiting that somewhat by listing the organizations, to exclude specs in personal repos.

I'm worried that trying to spell out the criteria for each organization will be messy, but I might be wrong. Could you suggest what it would be for W3C, for example?

**Comment by foolip** [comment]
Looking at https://github.com/w3c/browser-specs can help show the set of specifications we'd be looking to categorize into standards track and not.

Here's the list of W3C Working Groups that appear in browser-specs:

- Accessible Rich Internet Applications Working Group
- Audio Working Group
- Browser Testing and Tools Working Group
- Cascading Style Sheets (CSS) Working Group
- Devices and Sensors Working Group
- EPUB 3 Working Group
- GPU for the Web Working Group
- HTML Media Extensions Working Group
- Immersive Web Working Group
- Internationalization Working Group
- JSON-LD Working Group
- Math Working Group
- Media Working Group
- MiniApps Working Group
- Pointer Events Working Group
- Portable Network Graphics (PNG) Working Group
- SVG Working Group
- Second Screen Working Group
- Service Workers Working Group
- Timed Text Working Group
- Tracking Protection Working Group
- Web Application Security Working Group
- Web Applications Working Group
- Web Authentication Working Group
- Web Cryptography Working Group
- Web Editing Working Group
- Web Events Working Group
- Web Fonts Working Group
- Web Machine Learning Working Group
- Web Payments Working Group
- Web Performance Working Group
- Web Platform Working Group
- Web Real-Time Communications Working Group
- WebAssembly Working Group
- WebTransport Working Group

And these are the groups that _aren't_ Working Groups:

- Advisory Board
- Patents and Standards Interest Group
- Privacy Community Group
- Privacy Interest Group
- Technical Architecture Group
- W3C Process Community Group 
- Web Bluetooth Community Group
- Web NFC Community Group
- Web Platform Incubator Community Group
- WebAssembly Community Group

**Comment by foolip** [comment]
And these are all of the "specs" (some really aren't) from W3C that aren't from a Working Group:

- [Accelerated Shape Detection in Images](https://wicg.github.io/shape-detection-api/)
- [Accelerated Text Detection in Images](https://wicg.github.io/shape-detection-api/text.html)
- [Attribution Reporting](https://wicg.github.io/attribution-reporting-api/)
- [Background Fetch](https://wicg.github.io/background-fetch/)
- [Background Sync](https://wicg.github.io/background-sync/spec/)
- [Badging API](https://w3c.github.io/badging/)
- [Capability Delegation](https://wicg.github.io/capability-delegation/spec.html)
- [Client Hints Infrastructure](https://wicg.github.io/client-hints-infrastructure/)
- [Close Watcher API](https://wicg.github.io/close-watcher/)
- [Compression Streams](https://wicg.github.io/compression/)
- [Compute Pressure 1](https://wicg.github.io/compute-pressure/)
- [Contact Picker API](https://w3c.github.io/contact-api/spec/)
- [Content Index](https://wicg.github.io/content-index/spec/)
- [Cookie Store API](https://wicg.github.io/cookie-store/)
- [Crash Reporting](https://wicg.github.io/crash-reporting/)
- [CSS Parser API](https://wicg.github.io/css-parser-api/)
- [Custom State Pseudo Class](https://wicg.github.io/custom-state-pseudo-class/)
- [DataCue API](https://wicg.github.io/datacue/)
- [Deprecation Reporting](https://wicg.github.io/deprecation-reporting/)
- [Digital Goods API](https://wicg.github.io/digital-goods/)
- [Document Policy](https://wicg.github.io/document-policy/)
- [Element Timing API](https://wicg.github.io/element-timing/)
- [EyeDropper API](https://wicg.github.io/eyedropper-api/)
- [Federated Credential Management API](https://fedidcg.github.io/FedCM/)
- [File and Directory Entries API](https://wicg.github.io/entries-api/)
- [File System Access](https://wicg.github.io/file-system-access/)
- [Get Installed Related Apps API](https://wicg.github.io/get-installed-related-apps/spec/)
- [HTML Sanitizer API](https://wicg.github.io/sanitizer-api/)
- [HTMLVideoElement.requestVideoFrameCallback()](https://wicg.github.io/video-rvfc/)
- [Idle Detection API](https://wicg.github.io/idle-detection/)
- [Import Maps](https://wicg.github.io/import-maps/)
- [Ink API](https://wicg.github.io/ink-enhancement/)
- [Input Device Capabilities](https://wicg.github.io/input-device-capabilities/)
- [Intervention Reporting](https://wicg.github.io/intervention-reporting/)
- [isInputPending](https://wicg.github.io/is-input-pending/)
- [JS Self-Profiling API](https://wicg.github.io/js-self-profiling/)
- [Keyboard Lock](https://wicg.github.io/keyboard-lock/)
- [Keyboard Map](https://wicg.github.io/keyboard-map/)
- [Layout Instability](https://wicg.github.io/layout-instability/)
- [Loading Signed Exchanges](https://wicg.github.io/webpackage/loading.html)
- [Local Font Access API](https://wicg.github.io/local-font-access/)
- [Manifest Incubations](https://wicg.github.io/manifest-incubations/)
- [Measure Memory API](https://wicg.github.io/performance-measure-memory/)
- [Media Feeds](https://wicg.github.io/media-feeds/)
- [Mitigating Browser Fingerprinting in Web Specifications](https://w3c.github.io/fingerprinting-guidance/)
- [Navigation API](https://wicg.github.io/navigation-api/)
- [Network Information API](https://wicg.github.io/netinfo/)
- [overscroll and scrollend events](https://wicg.github.io/overscroll-scrollend-events/)
- [Page Lifecycle](https://wicg.github.io/page-lifecycle/)
- [Portals](https://wicg.github.io/portals/)
- [preferCurrentTab](https://wicg.github.io/prefer-current-tab/)
- [Prefetch](https://wicg.github.io/nav-speculation/prefetch.html)
- [Prerendering Revamped](https://wicg.github.io/nav-speculation/prerendering.html)
- [Prioritized Task Scheduling](https://wicg.github.io/scheduling-apis/)
- [Priority Hints](https://wicg.github.io/priority-hints/)
- [Private Click Measurement](https://privacycg.github.io/private-click-measurement/)
- [Private Network Access](https://wicg.github.io/private-network-access/)
- [Relinquishing Permissions](https://wicg.github.io/permissions-revoke/)
- [Requesting Permissions](https://wicg.github.io/permissions-request/)
- [Responsive Image Client Hints](https://wicg.github.io/responsive-image-client-hints/)
- [Save Data API](https://wicg.github.io/savedata/)
- [Scripting Policy](https://wicg.github.io/csp-next/scripting-policy.html)
- [Secure Curves in the Web Cryptography API](https://wicg.github.io/webcrypto-secure-curves/)
- [SMS One-Time Codes](https://wicg.github.io/sms-one-time-codes/)
- [Speculation Rules](https://wicg.github.io/nav-speculation/speculation-rules.html)
- [Text Fragments](https://wicg.github.io/scroll-to-text-fragment/)
- [The Storage Access API](https://privacycg.github.io/storage-access/)
- [URLPattern API](https://wicg.github.io/urlpattern/)
- [User Preference Media Features Client Hints Headers](https://wicg.github.io/user-preference-media-features-headers/)
- [User-Agent Client Hints](https://wicg.github.io/ua-client-hints/)
- [W3C Patent Policy](https://www.w3.org/Consortium/Patent-Policy/)
- [W3C Process Document](https://www.w3.org/Consortium/Process/)
- [Web Bluetooth](https://webbluetoothcg.github.io/web-bluetooth/)
- [Web NFC API](https://w3c.github.io/web-nfc/)
- [Web Periodic Background Synchronization](https://wicg.github.io/periodic-background-sync/)
- [Web Serial API](https://wicg.github.io/serial/)
- [Web Share Target API](https://w3c.github.io/web-share-target/)
- [Web Speech API](https://wicg.github.io/speech-api/)
- [WebAssembly JavaScript Interface: Exception Handling](https://webassembly.github.io/exception-handling/js-api/)
- [WebHID API](https://wicg.github.io/webhid/)
- [WebOTP API](https://wicg.github.io/web-otp/)
- [WebUSB API](https://wicg.github.io/webusb/)
- [Window Controls Overlay](https://wicg.github.io/window-controls-overlay/)

**Comment by karlcow** [comment]
> And these are all of the "specs" (some really aren't) from W3C that aren't from a Working Group:

Yes in this list, a lot of these have different stances from different browsers and It's probably better if these were not part of the effort until the relevant standard organization has resolved their status on being on a standard track.

**Comment by foolip** [comment]
I agree, most of these would have a very low chance of being accepted as proposals due to a lack of implementer interest. I think this is also the case for plenty of Working Group specs as well. Implementation status or interest seems like the best predictor, not standards group status.

 Some of CG specs listed _might_ be reasonable proposals though:

- [HTML Sanitizer API](https://wicg.github.io/sanitizer-api/) (Google+Mozilla editors, Gecko implementing)
- [overscroll and scrollend events](https://wicg.github.io/overscroll-scrollend-events/) (shipped in Chrome, could make sense in our Scrolling focus area)
- [SMS One-Time Codes](https://wicg.github.io/sms-one-time-codes/) (Apple+Google editors)
- [The Storage Access API](https://privacycg.github.io/storage-access/) (Apple+Google+Mozilla editors)
- [Web Speech API](https://wicg.github.io/speech-api/) (parts shipping everywhere, Google+Mozilla editors)

I understand the desire to make the conversation very short if we get a proposal for a feature that only Google is interested in right now, but hoping for a charter that doesn't treat it as exceptional if all want to work together on something that's currently in a CG.

**Comment by fantasai** [comment]
For W3C stuff, if something has significant implementer interest, it should chartered into a Working Group. If that's not happening Interop should push for that to happen rather than letting things stay in CGs way past their “incubation” phase due to inertia.

**Comment by foolip** [comment]
This is now defined in https://github.com/web-platform-tests/interop/pull/115. @karlcow can you review that? We might want to put it in some README as well, but I wasn't sure where would make sense.

**Comment by foolip** [comment]
We put this https://github.com/web-platform-tests/interop/pull/115, closing.
